app/assets/stylesheets/custom.scss

```diff
@@ -91,3 +91,83 @@ footer {
     }
   }
 }
+
+@mixin box_sizing {
+  -moz-box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+.debug_dump {
+  clear: both;
+  float: left;
+  width: 100%;
+  margin-top: 45px;
+  @include box_sizing;
+}
+
+/* sidebar */
+aside {
+  section.user_info {
+    margin-top: 20px;
+  }
+
+  section {
+    padding: 10px 0;
+    margin-top: 20px;
+    &:first-child {
+      border: 0;
+      padding-top: 0;
+    }
+
+    span {
+      display: block;
+      margin-bottom: 3px;
+      line-height: 1;
+    }
+
+    h1 {
+      font-size: 1.4em;
+      text-align: left;
+      letter-spacing: -1px;
+      margin-bottom: 3px;
+      margin-top: 0px;
+    }
+  }
+}
+
+.gravatar {
+  float: left;
+  margin-right: 10px;
+}
+
+.gravatar_edit {
+  margin-top: 15px;
+}
+
+/* forms */
+input, textarea, select, .uneditable-input {
+  border: 1px solid $bbb;
+  width: 100%;
+  margin-bottom: 15px;
+  @include box_sizing;
+}
+
+input {
+  height: auto !important;
+}
```

Comment:
Những dòng css có tác động scope global như này thường tránh sử dụng, bạn sử dụng thì miễn bạn nắm rõ là oke, cơ mà những code global thẳng vào tag html này sẽ luôn được đặt lên đầu file hoặc tách riêng file để quản lý ạ... Cái này là kinh nghiệm sau này nhaa :D

Reply:
nghia là mình lên chuyển nó nên sau các dòng @import  đúng không ạ

Reply:
Dạ không, nếu bạn tách file các css tác động thẳng global tags thì càng tốt, xong @import vô cũng được. Còn không bạn cho hết lên đầu file này nha, bạn để ý các code của các frameworkss lớn hoặc projects lớn họ đều làm vậy ạ. Vì code thằng css này mà quản lý không tốt tới lúc phình to ra, các css nó overrides nhau nảy sinh các vấn đề giao diện rất khó theo dõi và sửa :(

Reply:
Ví dụ code của trên của bạn, đúng là hiện tại có thể mình chỉ dùng với thằng forms, nhưng sau này nảy (với project khác to hơn) sẽ nảy sinh nhiều nhu cầu sử dụng các tag kia hơn chẳng hạn, thì khi đó những code mà ngày xưa mình code kiểu như trên sẽ vô hình chung tác động vào cái mới mình định làm, và khi mình sửa lại nó thì lại ảnh hưởng tới rất nhiều chỗ khác... Lời khuyên của mình là ở các tag global, mình chỉ nên css những thuộc tính có mức ảnh hưởng chung và ít thay đổi như font-size, font-weight, letter-spacing, display,... còn những cái liên quan tới kích thước, margin, padding, border, outline... thì tránh. Trừ khi mình chắc chắn là nhu cầu của mình là không thay đổi về sau và muốn thực hiện đồng hộ hoá tất cả các thành phần sử dụng tag đó sau này.... Sorry mình nói hơi dài dòng @@

Reply:
Sức mạnh của SCSS rất lớn, mình có thể khai báo class dùng chung cho các thành phần mình mong muốn và cần thì thêm class đó vô thôi thay vì code trực tiếp vào tag global... Trong SCSS mình còn có @extend, @include .... nữa mà :D

Reply:
oke minh cam on